src/js/providers/flash.js

```diff
@@ -348,18 +344,18 @@ define([
                     }
                     _container = null;
                     _item = null;
-                    _eventDispatcher.resetEventListeners();
-                    _eventDispatcher = null;
+                    this.off();
                 }
         });
 
         // Overwrite the event dispatchers to block on certain occasions
-        this.sendEvent = function() {
+        this.sendEvent = function(type, data) {
```

Comment:
Let's not use `sendEvent` anymore since it can be confusing if both `sendEvent` and `trigger` can be used on an object. You can do:
```
this.trigger = function(type, args) {
    if (!_attached) {
        return;
    }
    return Events.trigger.call(this, type, args);
};
```